about/policy/data-sharing-agreement/index.md

```diff
@@ -0,0 +1,69 @@
+
+`<Four different levels of getting agreement/acknowledgement, from easiest to most onerous>`
+
+In order to ~~emulate the automatic EULAs of big, powerful software companies~~ reduce the friction of adding data to the SGID, by completing a [Porter issue](https://github.com/agrc/porter/issues) to submit your data to the SGID, you agree to this Data Sharing Agreement.
+
+—OR—
+
+You accept this Data Sharing Agreement by checking the appropriate checkbox in the [Porter issue](https://github.com/agrc/porter/issues) tracking your SGID data submission.
+
+—OR—
+
+You must visit `<Google Form/other web-based form and response tracker>` and agree to this Data Sharing Agreement prior to creating a new [Porter issue](https://github.com/agrc/porter/issues) to submit your data to the SGID.
+
+—OR—
+
+You must submit a signed copy of this Data Sharing Agreement to our office prior to creating a new [Porter issue](https://github.com/agrc/porter/issues) to submit your data to the SGID.
+
+## Definitions
+{: .text-left}
+
+**"Aggregate data"** refers to any data that cover a specific area of the state and are provided by another organization (such as a city or county) for UGRC to combine with other similar data into statewide datasets.
+
+**"External data"** refers to any data stored or hosted by a person or organization outside of UGRC.
+
+**"Hosted data"** refers to any data stored, maintained, and served by UGRC, whether created by UGRC or provided by another organization. It may include aggregate data.
+
+**"SGID entry"** refers to any of the following:
+
+1. Links in the SGID Index to external data
+1. Links and references in opendata.gis.utah.gov to external data in the form of ArcGIS Online items owned by other organizations
+1. Hosted data in the Open SGID
+1. Hosted data in UGRC's ArcGIS Online organization that are shared with an SGID ArcGIS Online group or the "AGRC Shelf" group
+
+The **"steward"** refers to the individual or organization that shares data with UGRC for inclusion in the SGID.
+
+## Access and Data Quality Provisions
+{: .text-left}
+
+- UGRC will make all SGID entries available to the general public without any form of login or account.
+- UGRC may edit any hosted data to fix any technical errors that prevent the data from being stored in the relevant databases.
+- UGRC may edit any aggregate data to conform with established standards and schemas for statewide datasets.
+- The steward will ensure any links or connections to external data remain live and do not require any form of login or account.
+- The steward will take reasonable efforts to not modify the schema of all external data, with the exception of adding new fields.
+- The steward may request UGRC to remove their SGID entries at any time. UGRC will comply with these requests as quickly as possible through the normal [dataset deprecation process](https://github.com/agrc/porter/blob/main/.github/ISSUE_TEMPLATE/2-deprecate-sgid-dataset.md).
```

Comment:
**thought**: this sounds like it could be problematic if the data is highly utilized

Reply:
(assuming you're talking about removal requests) My goal was to formalize our current behavior. Is this text too encouraging about dataset removal? Maybe something like 'may request to remove if necessary'?

Reply:
yeh what if someone gets mad and wants to remove land ownership or something. It seems like a [left-pad npm like issue](https://blog.npmjs.org/post/141577284765/kik-left-pad-and-npm)

Reply:
woah, that was a big mess. How does the new language strike you? The link anchor is for a new policy page (an older but more-or-less the same version is in the old PR at https://deploy-preview-1627--gis-utah-gov.netlify.app/about/policy/sgid/#reasons-for-removal).

Reply:
I've made another edit; now links to the full removal section.